deployment/components/topology-updater/topologyupdater-mounts.yaml

```diff
@@ -21,10 +21,6 @@
   - name: host-sys
     mountPath: /host-sys
 
-- op: add
-  path: /spec/template/spec/containers/0/args/-
-  value: "-kubelet-config-file=/host-var/lib/kubelet/config.yaml"
```

Comment:
Here we need to align the Helm chart, too, I think

Reply:
I though it has already aligned, the helm chart does not have a `--kubelet-config-file` parameters in `deployment/helm/node-feature-discovery/templates/topologyupdater.yaml`.

Reply:
Ach, you're right